20-motor-monitor-and-vibration/README.md

```diff
@@ -34,13 +37,18 @@ send alerts when the vibration is not within the expected range for the motor's
 * (optional - for power outage events) [LiPo battery with JST cable](https://shop.blues.io/products/5-000-mah-lipo-battery)
 * [Notecarrier-B](https://shop.blues.io/products/carr-b)
 * [Notecard](https://blues.io/products/notecard/)
-* A switched motor that has a 3v3 input or output that controls or indicates activity
+* A switched motor that has a 3v3 input or output that controls or indicates activity. (See the next section for details.)
 * A pair of jumper cables
 
+## Motor Choice and Testing
+
+This project illustrates an app that monitors an industrial motor for activity and vibration and can in principle operate with almost any kind of motor. In an industrial setting this would typically be a an electric motor already installed at a facility. In addition to the motor, a 3.3-5v logic signal is used to indicate when the motor is active. In cases where this is not already present, it can be retrofitted using an optocoupler placed in parallel with the motor's electrical supply on the input side, which switches a 3.3v line from the Notecarrier on the output side, indicating when the motor is active.
```

Comment:
```suggestion
This project illustrates an app that monitors an industrial motor for activity and vibration, and can in principle operate with almost any kind of motor. In an industrial setting this would typically be an electric motor already installed at a facility. In addition to the motor, a 3.3-5v logic signal is used to indicate when the motor is active. In cases where this is not already present, it can be retrofitted using an optocoupler placed in parallel with the motor's electrical supply on the input side, which switches a 3.3v line from the Notecarrier on the output side, indicating when the motor is active.
```